.github/CHANGELOG.md

```diff
@@ -1,11 +1,35 @@
-## Release 0.6.0a3 (development release)
+## Release 0.6.1a3 (development release)
 
 ### New features since the last release
 
 ### Bug fixes
 
 ### Improvements
 
+### Documentation changes
+
+### Contributors
+
+This release contains contributions from (in alphabetical order):
+
+See full commit details ...
+
+---
+
+
+## Release 0.6.1a3 (current release)
+
+### New features since the last release
+
+* Fixed drawing of stabilizer graph for zero syndrome: [(#9 | compatible feature)](https://github.com/XanaduAI/flamingpy/pull/9)
```

Comment:
@ilan-tz and @maxtremblay, are these really breaking or backward compatible? My assumption was "breaking", but got rather unsure after experimenting some more with them.

Reply:
They break the api. In fact, during that PR we changed the stabilizer graph api